edge/pkg/metamanager/process.go

```diff
@@ -238,11 +239,30 @@ func (m *metaManager) processDelete(message model.Message) {
 	sendToCloud(resp)
 }
 
+// KeyFunc keys should be nonconfidential and safe to log
+func KeyFunc(resource string, tr *authenticationv1.TokenRequest) string {
+	return fmt.Sprintf("%s/%s/%s/%s", resource, tr.Spec.BoundObjectRef.Kind, tr.Spec.BoundObjectRef.Name, tr.Spec.BoundObjectRef.UID)
```

Comment:
In kubernetes， the key is
```
func keyFunc(name, namespace string, tr *authenticationv1.TokenRequest) string {
	var exp int64
	if tr.Spec.ExpirationSeconds != nil {
		exp = *tr.Spec.ExpirationSeconds
	}

	var ref authenticationv1.BoundObjectReference
	if tr.Spec.BoundObjectRef != nil {
		ref = *tr.Spec.BoundObjectRef
	}

	return fmt.Sprintf("%q/%q/%#v/%#v/%#v", name, namespace, tr.Spec.Audiences, exp, ref)
}
```

if user update the token request info in pod, the token is different
```
  volumes:
  - name: istio-token
    projected:
      defaultMode: 420
      sources:
      - serviceAccountToken:
          audience: istio-ca
          expirationSeconds: 43200
          path: istio-token
```

so we should consider the key func to adapt to these cases

Reply:
`return fmt.Sprintf("%q/%#v/%#v/%#v", resource, tr.Spec.Audiences, exp, ref.UID)`
The max length of `key` in sqlite db is  256, minimize the `key` length by store `ref.UID`.